supertokens_python/recipe/thirdparty/api/implementation.py

```diff
@@ -44,11 +44,13 @@ async def authorisation_url_get(
         self,
         provider: Provider,
         redirect_uri_on_provider_dashboard: str,
+        tenant_id: str,
         api_options: APIOptions,
         user_context: Dict[str, Any],
     ) -> Union[AuthorisationUrlGetOkResult, GeneralErrorResponse]:
         authorisation_url_info = await provider.get_authorisation_redirect_url(
             redirect_uri_on_provider_dashboard=redirect_uri_on_provider_dashboard,
+            tenant_id=tenant_id,
```

Comment:
node SDK does not take tenant_id for this function - why does the python SDK need it?

Reply:
Removed